modules/arm_plugin/src/arm_converter/arm_converter.cpp

```diff
@@ -193,7 +193,7 @@ Converter::Converter(const std::shared_ptr<const ngraph::Function> function, boo
             auto hasQuantizationInfo = (itInfo != rt_info.end());
             arm_compute::TensorInfo tensorInfo;
             if (quantizedOutput && hasQuantizationInfo) {
-                auto& quantizationInfo = std::dynamic_pointer_cast<ngraph::VariantImpl<arm_compute::QuantizationInfo>>(itInfo->second)->get();
+                auto& quantizationInfo = ov::as_type_ptr<ngraph::VariantWrapper<arm_compute::QuantizationInfo>>(itInfo->second)->get();
```

Comment:
Shouldn't it be `safe_cast` as well?

Reply:
done

Reply:
Looks like it's still `as_type_ptr` and unaligned with `arm_converter_conv.cpp` where `safe_cast` is used

Reply:
@terfendail  Please have a look again